admin-api-server/test/app.e2e.spec.ts

```diff
@@ -2007,6 +2007,8 @@ describe('AppController (e2e)', () => {
             currency: 'EUR',
             edition_size: 1,
             transaction_value: '0.82',
+            fee_in_base_currency: null,
```

Comment:
I see your point about having difficulty making this part function well. We should write an additional function like `emulateNftSale` that fills the relevant que pasa tables with fake data as well, or amend `emulateNftSale` itself.

If you have time to do this, great if it can be done for this PR. Otherwise I am also happy to delay this task for a next PR, ie to be picked up as a task later this week in that case (by either you or me).